src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -2669,6 +2576,8 @@ MavlinkReceiver::receive_thread(void *arg)
 	hrt_abstime last_send_update = 0;
 
 	while (!_mavlink->_task_should_exit) {
+		update_params(false);
```

Comment:
@dagar, hopefully this what you had in mind, (see method below)?

Reply:
Yes this should do.